styles/styles.css

```diff
@@ -356,3 +369,159 @@ footer {
   background: #0d151e;
   border-radius: 24px;
 }
+
+@media screen and (min-width: 768px) {
+  .toolbar {
+    display: none;
+  }
+
+  .toolbar-desktop {
+    display: block;
+  }
+
+  .toolbar-desktop ul {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    list-style: none;
+    height: 72px;
+    font-size: 15px;
+  }
+
+  .toolbar-desktop ul > * {
+    margin: 0 10px;
+  }
+
+  a:link {
+    text-decoration: none;
+  }
+
+  a:visited {
+    text-decoration: none;
+  }
+
+  a:hover {
+    text-decoration: none;
+  }
+
+  a:active {
+    text-decoration: none;
+  }
+
+  .toolbar-desktop ul > * a {
+    color: #344563;
+  }
+
+  header {
+    background-image: url(../media/Header-llustration-desktop.png);
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    background-position: 0;
+    background-size: cover;
+  }
```

Comment:
- [ ] Kindly make sure that your design looks fine and matches the Figma design on width `768px`, [as per requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md#:~:text=(Desktop%3A%20768px%20or%20wider).).

- On width `900px` to `768px`, The project cards don't get any smaller and they start taking more width than they should, which leads to the following behavior.
![chrome-capture-2022-2-31](https://user-images.githubusercontent.com/62290545/161150380-f40c852d-30c0-478f-9ca8-b9128df520fc.gif)
![image](https://user-images.githubusercontent.com/62290545/161150559-6e9e38d7-28d6-4544-a478-fd3a51b38a9c.png)

[Hint] You can easily fix this by using `minmax` for the `grid-template-columns` property. 
https://github.com/Dredwell/my-portfolio/blob/1cabc16c7d32cf13d43eae70b6303ea87fe5f248/styles/styles.css#L445
You can read more about this [here](https://css-tricks.com/you-want-minmax10px-1fr-not-1fr/).
After doing so, kindly fix the display of technologies for each project card on width `768px`.